### ISSUE TITLE
provide anonymous user for newcommers in Eclipse Che / Hosted Che

### ISSUE THREAD

**Issue by gattytto** [eclipse-che/che#18306]
some people (like me) just don't like having to create an account for the single purpose of testing something. some even don't have a secondary e-mail account to address the spam generated by those accounts that may be used only once. 

This seems like too much work to try for once. Some sites have anonxxyy users available in case one just wants to "see" or for example, if I need help from developers in a specific language community and I need them to provide me a hint for fixing a demo project to include in a stack or some other single-usage task it would be nice to have a factory link with an "anonymous" additional argument so those users don't need to create an account to start a workspace. 

in the case they want to keep using it then they could click some button and create an account that persists the workspaces they created in the anonymous account, otherwise the anonymous account content could be deleted in 24hs or something.

**Comment by ibuziuk** [comment]
@gattytto this feature makes sense, but we do not have capacity to implement it at the moment. 
Will you be interested in contributing the anonymous user feature in the upstream?

**Comment by gattytto** [comment]
I will research on the keycloak anonymous feature so maybe an ephemereal user can be allocated if no cookie is set (instead of redirecting to login landing).
